ToyBox/ToyBox.csproj

```diff
@@ -315,7 +318,7 @@
   </PropertyGroup>
   <ProjectExtensions>
     <VisualStudio>
-      <UserProperties info_1json__JsonSchema="https://json.schemastore.org/global.json" config_4bindings_1json__JsonSchema="https://json.schemastore.org/global.json" />
+      <UserProperties config_4bindings_1json__JsonSchema="https://json.schemastore.org/global.json" info_1json__JsonSchema="https://json.schemastore.org/global.json" />
```

Comment:
No clue what happened here or why. Looks like the order changed for some reason. I didn't make this change.

Reply:
I dunno why this happens.  ADDB hits the same issue.

Reply:
...and it changed it back. Weird.